docs/docs/examples/login/vue.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Vue
+---
+
+This is our Zitadel [Vue.js](https://vuejs.org/) template. It shows how to authenticate as a user and show user information.
+
+![Vue Screenshot](/img/vue/app-screen.png)
+
+## Getting Started
+
+First, we start by creating a new Vue app with `create-vue`, the official project scaffolding tool, which sets up everything automatically for you. To create a project, run:
+
+```bash
+npm init vue@latest
+```
+
+## Install Authentication library
+
+To keep the template as easy as possible we use [vue-oidc-client](https://github.com/soukoku/vue-oidc-client) as our main authentication library. To install, run:
+
+```bash
```

Comment:
Since this whole file is redundant with the Zitadel-example repo I would recommend linking towards the docs and remove this content from the source

Reply:
@fforootd wouldn't be the sane solution to link the code snippets from the repo?
internally this would work with docusaurus natively (https://docusaurus.io/docs/markdown-features/code-blocks#code-title) but maybe there's a good plugin (https://github.com/saucelabs/docusaurus-theme-github-codeblock)

Reply:
Looks interesting need to investigate